### DISCUSSION TITLE
Enable Push Notifications Capacitor

### DISCUSSION THREAD

**Discussion by serascandia** [quasarframework/quasar#13145] [CLI - Capacitor / Cordova modes] [answered]
Hello community hope you can help me, I try to integrate push notifications into my app I use capacitor maybe some of you have a guide to do it, I try to follow some I found in internet but not luck.   I have errors in xcode like this.

Type 'NSNotification.Name' has no member 'capacitorDidFailToRegisterForRemoteNotifications'

it is appreciated

**Comment by ontwikkelfabriek** [comment]
@serascandia 

Here's what I could find, but you should be asking these questions at the Capacitor community.

Article:
https://dev.to/marcoimproda90/how-to-configure-firebase-push-notifications-with-ionic-vue-js-38n7

Repo of demo app:
https://github.com/marcoimproda/ionic-vue-push-notifications-demo

Capacitor docs:
https://capacitorjs.com/docs/apis/push-notifications

> Type 'NSNotification.Name' has no member 'capacitorDidFailToRegisterForRemoteNotifications'

https://forum.ionicframework.com/t/error-push-notifications-with-firebase-in-an-ionic-version-ios/211268

**Reply by serascandia** [reply]
Thanks

**Comment by serascandia** [comment, marked as answer]
after a while struggling with this, I found a module to work with capacitor 3.0, I hope this helps to other users.

https://github.com/robingenz/capacitor-firebase
